src/components/views/room_settings/RelatedGroupSettings.js

```diff
@@ -110,6 +110,7 @@ module.exports = React.createClass({
                 items={this.state.newGroupsList}
                 className={"mx_RelatedGroupSettings"}
                 newItem={this.state.newGroupId}
+                canEdit={this.props.canSetRelatedRooms}
```

Comment:
I probably missed this in an earlier PR. but should this be `canSetRelatedGroups`?

Reply:
yep!